Sync WPT with upstream (25-04-2018)

servo-wpt-sync opened the pull request:
:warning: Do not merge this PR without verifying that it is not overwriting local changes to web-platform-tests. :warning:

Automated downstream sync of changes from upstream as of 25-04-2018.
[no-wpt-sync]

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/servo/servo/20693)
<!-- Reviewable:end -->

Commit:
Update web-platform-tests to revision 5084587f6b05bf99ad09e7844be66dcc61070cdf

nox commented:
@bors-servo r+ p=100

bors-servo commented:
:pushpin: Commit 18c600d has been approved by `nox`

<!-- @bors-servo r=nox 18c600ded778ea5250c1f4e3d4b8f94f8a803e7f -->

bors-servo commented:
:hourglass: Testing commit 18c600ded778ea5250c1f4e3d4b8f94f8a803e7f with merge a77cf32b6d2cad67d21c91ab4baec895c84ce31e...

bors-servo commented:
:broken_heart: Test failed - [linux-dev](http://build.servo.org/builders/linux-dev/builds/11706)

nox commented:
@bors-servo r+

bors-servo commented:
:bulb: This pull request was already approved, no need to approve it again.

- This pull request previously failed. You should add more commits to fix the bug, or use `retry` to trigger a build again.

bors-servo commented:
:pushpin: Commit 18c600d has been approved by `nox`

<!-- @bors-servo r=nox 18c600ded778ea5250c1f4e3d4b8f94f8a803e7f -->

bors-servo commented:
:hourglass: Testing commit 18c600ded778ea5250c1f4e3d4b8f94f8a803e7f with merge 7d0be251a071d11e65feb495b563506363782766...

bors-servo commented:
:broken_heart: Test failed - [linux-dev](http://build.servo.org/builders/linux-dev/builds/11707)

nox committed:
Bump html5ever to 0.2.23 for the stack size fix

bors-servo commented:
:broken_heart: Test failed - [linux-rel-wpt](http://build.servo.org/builders/linux-rel-wpt/builds/8607)

nox commented:
@bors-servo treeclosed+ p=9001

nox commented:
@bors-servo retry

bors-servo commented:
:zap: Previous build results for [android](http://build.servo.org/builders/android/builds/11395), [arm32](http://build.servo.org/builders/arm32/builds/11626), [arm64](http://build.servo.org/builders/arm64/builds/11672), [linux-dev](http://build.servo.org/builders/linux-dev/builds/11708), [linux-rel-css](http://build.servo.org/builders/linux-rel-css/builds/8604), [mac-dev-unit](http://build.servo.org/builders/mac-dev-unit/builds/11761), [mac-rel-css1](http://build.servo.org/builders/mac-rel-css1/builds/4157), [mac-rel-wpt1](http://build.servo.org/builders/mac-rel-wpt1/builds/8429), [mac-rel-wpt2](http://build.servo.org/builders/mac-rel-wpt2/builds/8491), [mac-rel-wpt3](http://build.servo.org/builders/mac-rel-wpt3/builds/4252), [mac-rel-wpt4](http://build.servo.org/builders/mac-rel-wpt4/builds/4193), [windows-msvc-dev](http://build.servo.org/builders/windows-msvc-dev/builds/7100) are reusable. Rebuilding only [linux-rel-wpt](http://build.servo.org/builders/linux-rel-wpt/builds/8607), [mac-rel-css2](http://build.servo.org/builders/mac-rel-css2/builds/4181)...

bors-servo commented:
:broken_heart: Test failed - [mac-rel-css2](http://build.servo.org/builders/mac-rel-css2/builds/4182)

nox commented:
@bors-servo retry

bors-servo commented:
:zap: Previous build results for [android](http://build.servo.org/builders/android/builds/11395), [arm32](http://build.servo.org/builders/arm32/builds/11626), [arm64](http://build.servo.org/builders/arm64/builds/11672), [linux-dev](http://build.servo.org/builders/linux-dev/builds/11708), [linux-rel-css](http://build.servo.org/builders/linux-rel-css/builds/8604), [linux-rel-wpt](http://build.servo.org/builders/linux-rel-wpt/builds/8608), [mac-dev-unit](http://build.servo.org/builders/mac-dev-unit/builds/11761), [mac-rel-css1](http://build.servo.org/builders/mac-rel-css1/builds/4157), [mac-rel-wpt1](http://build.servo.org/builders/mac-rel-wpt1/builds/8429), [mac-rel-wpt2](http://build.servo.org/builders/mac-rel-wpt2/builds/8491), [mac-rel-wpt3](http://build.servo.org/builders/mac-rel-wpt3/builds/4252), [mac-rel-wpt4](http://build.servo.org/builders/mac-rel-wpt4/builds/4193), [windows-msvc-dev](http://build.servo.org/builders/windows-msvc-dev/builds/7100) are reusable. Rebuilding only [mac-rel-css2](http://build.servo.org/builders/mac-rel-css2/builds/4182)...

bors-servo commented:
:broken_heart: Test failed - [mac-rel-css2](http://build.servo.org/builders/mac-rel-css2/builds/4183)

nox commented:
@bors-servo retry

bors-servo commented:
:zap: Previous build results for [android](http://build.servo.org/builders/android/builds/11395), [arm32](http://build.servo.org/builders/arm32/builds/11626), [arm64](http://build.servo.org/builders/arm64/builds/11672), [linux-dev](http://build.servo.org/builders/linux-dev/builds/11708), [linux-rel-css](http://build.servo.org/builders/linux-rel-css/builds/8604), [linux-rel-wpt](http://build.servo.org/builders/linux-rel-wpt/builds/8608), [mac-dev-unit](http://build.servo.org/builders/mac-dev-unit/builds/11761), [mac-rel-css1](http://build.servo.org/builders/mac-rel-css1/builds/4157), [mac-rel-wpt1](http://build.servo.org/builders/mac-rel-wpt1/builds/8429), [mac-rel-wpt2](http://build.servo.org/builders/mac-rel-wpt2/builds/8491), [mac-rel-wpt3](http://build.servo.org/builders/mac-rel-wpt3/builds/4252), [mac-rel-wpt4](http://build.servo.org/builders/mac-rel-wpt4/builds/4193), [windows-msvc-dev](http://build.servo.org/builders/windows-msvc-dev/builds/7100) are reusable. Rebuilding only [mac-rel-css2](http://build.servo.org/builders/mac-rel-css2/builds/4183)...

bors-servo commented:
:broken_heart: Test failed - [mac-rel-css2](http://build.servo.org/builders/mac-rel-css2/builds/4184)

nox commented:
@bors-servo retry

bors-servo commented:
:zap: Previous build results for [android](http://build.servo.org/builders/android/builds/11395), [arm32](http://build.servo.org/builders/arm32/builds/11626), [arm64](http://build.servo.org/builders/arm64/builds/11672), [linux-dev](http://build.servo.org/builders/linux-dev/builds/11708), [linux-rel-css](http://build.servo.org/builders/linux-rel-css/builds/8604), [linux-rel-wpt](http://build.servo.org/builders/linux-rel-wpt/builds/8608), [mac-dev-unit](http://build.servo.org/builders/mac-dev-unit/builds/11761), [mac-rel-css1](http://build.servo.org/builders/mac-rel-css1/builds/4157), [mac-rel-wpt1](http://build.servo.org/builders/mac-rel-wpt1/builds/8429), [mac-rel-wpt2](http://build.servo.org/builders/mac-rel-wpt2/builds/8491), [mac-rel-wpt3](http://build.servo.org/builders/mac-rel-wpt3/builds/4252), [mac-rel-wpt4](http://build.servo.org/builders/mac-rel-wpt4/builds/4193), [windows-msvc-dev](http://build.servo.org/builders/windows-msvc-dev/builds/7100) are reusable. Rebuilding only [mac-rel-css2](http://build.servo.org/builders/mac-rel-css2/builds/4184)...

nox commented:
@bors-servo retry

bors-servo commented:
:zap: Previous build results for [android](http://build.servo.org/builders/android/builds/11395), [arm32](http://build.servo.org/builders/arm32/builds/11626), [arm64](http://build.servo.org/builders/arm64/builds/11672), [linux-dev](http://build.servo.org/builders/linux-dev/builds/11708), [linux-rel-css](http://build.servo.org/builders/linux-rel-css/builds/8604), [linux-rel-wpt](http://build.servo.org/builders/linux-rel-wpt/builds/8608), [mac-dev-unit](http://build.servo.org/builders/mac-dev-unit/builds/11761), [mac-rel-css1](http://build.servo.org/builders/mac-rel-css1/builds/4157), [mac-rel-wpt1](http://build.servo.org/builders/mac-rel-wpt1/builds/8429), [mac-rel-wpt2](http://build.servo.org/builders/mac-rel-wpt2/builds/8491), [mac-rel-wpt3](http://build.servo.org/builders/mac-rel-wpt3/builds/4252), [mac-rel-wpt4](http://build.servo.org/builders/mac-rel-wpt4/builds/4193), [windows-msvc-dev](http://build.servo.org/builders/windows-msvc-dev/builds/7100) are reusable. Rebuilding only [mac-rel-css2](http://build.servo.org/builders/mac-rel-css2/builds/4184)...

bors-servo commented:
:broken_heart: Test failed - [mac-rel-css2](http://build.servo.org/builders/mac-rel-css2/builds/4186)

nox commented:
@bors-servo retry treeclosed+

bors-servo commented:
:zap: Previous build results for [android](http://build.servo.org/builders/android/builds/11395), [arm32](http://build.servo.org/builders/arm32/builds/11626), [arm64](http://build.servo.org/builders/arm64/builds/11672), [linux-dev](http://build.servo.org/builders/linux-dev/builds/11708), [linux-rel-css](http://build.servo.org/builders/linux-rel-css/builds/8604), [linux-rel-wpt](http://build.servo.org/builders/linux-rel-wpt/builds/8608), [mac-dev-unit](http://build.servo.org/builders/mac-dev-unit/builds/11761), [mac-rel-css1](http://build.servo.org/builders/mac-rel-css1/builds/4157), [mac-rel-wpt1](http://build.servo.org/builders/mac-rel-wpt1/builds/8429), [mac-rel-wpt2](http://build.servo.org/builders/mac-rel-wpt2/builds/8491), [mac-rel-wpt3](http://build.servo.org/builders/mac-rel-wpt3/builds/4252), [mac-rel-wpt4](http://build.servo.org/builders/mac-rel-wpt4/builds/4193), [windows-msvc-dev](http://build.servo.org/builders/windows-msvc-dev/builds/7100) are reusable. Rebuilding only [mac-rel-css2](http://build.servo.org/builders/mac-rel-css2/builds/4186)...

emilio commented:
@bors-servo treeclosed+

nox committed:
Don't clean nightlies on Mac builds

Everything is on fire on the CI because this fails to work on servo-mac5 for some
stupid reason. Let's avouid the issue for now because we need to be able to work.

nox commented:
@bors-servo r=nox,emilio

bors-servo commented:
:pushpin: Commit c937f9a has been approved by `nox,emilio`

<!-- @bors-servo r=nox,emilio c937f9a51a032c3e5d251ad10177af897cfe2673 -->

bors-servo commented:
:hourglass: Testing commit c937f9a51a032c3e5d251ad10177af897cfe2673 with merge 2daa923aaba572fb8b9cc7e19c9c46677b42159e...

nox commented:
@bors-servo r=nox,emilio

bors-servo commented:
:pushpin: Commit f35617f has been approved by `nox,emilio`

<!-- @bors-servo r=nox,emilio f35617fe26b6ea34ed5adeb040c08181e9899ded -->

bors-servo commented:
:hourglass: Testing commit f35617fe26b6ea34ed5adeb040c08181e9899ded with merge 498c7cb852988aaecdc0cc3c014c6cd6df88714d...

nox commented:
@bors-servo r=nox,emilio force p=9001 treeclosed=9000

bors-servo commented:
:pushpin: Commit d52011c has been approved by `nox,emilio`

<!-- @bors-servo r=nox,emilio d52011ceb396cc48817d8f2d8e6dd7ffe90e0aa2 -->

nox commented:
@bors-servo r=nox,emilio force

bors-servo commented:
:bulb: This pull request was already approved, no need to approve it again.

bors-servo commented:
:pushpin: Commit d52011c has been approved by `nox,emilio`

<!-- @bors-servo r=nox,emilio d52011ceb396cc48817d8f2d8e6dd7ffe90e0aa2 -->

nox commented:
@bors-servo treeclosed=1000 p=9000

nox commented:
@bors-servo retry clean force

nox commented:
Well, seems like closing the tree is broken too. Cc @edunham 

@bors-servo treeclosed-

bors-servo commented:
:hourglass: Testing commit d52011ceb396cc48817d8f2d8e6dd7ffe90e0aa2 with merge d234aeb5af5450702211511995b9a19c57e38d52...

bors-servo commented:
:broken_heart: Test failed - [mac-rel-wpt1](http://build.servo.org/builders/mac-rel-wpt1/builds/8433)

nox commented:
@bors-servo r+

bors-servo commented:
:bulb: This pull request was already approved, no need to approve it again.

- This pull request previously failed. You should add more commits to fix the bug, or use `retry` to trigger a build again.

bors-servo commented:
:pushpin: Commit d52011c has been approved by `nox`

<!-- @bors-servo r=nox d52011ceb396cc48817d8f2d8e6dd7ffe90e0aa2 -->

bors-servo commented:
:zap: Previous build results for [android](http://build.servo.org/builders/android/builds/11399), [arm32](http://build.servo.org/builders/arm32/builds/11630), [arm64](http://build.servo.org/builders/arm64/builds/11676), [linux-dev](http://build.servo.org/builders/linux-dev/builds/11712), [linux-rel-css](http://build.servo.org/builders/linux-rel-css/builds/8608), [linux-rel-wpt](http://build.servo.org/builders/linux-rel-wpt/builds/8612), [mac-dev-unit](http://build.servo.org/builders/mac-dev-unit/builds/11765), [mac-rel-css1](http://build.servo.org/builders/mac-rel-css1/builds/4161), [mac-rel-css2](http://build.servo.org/builders/mac-rel-css2/builds/4190), [mac-rel-wpt2](http://build.servo.org/builders/mac-rel-wpt2/builds/8495), [mac-rel-wpt3](http://build.servo.org/builders/mac-rel-wpt3/builds/4256), [mac-rel-wpt4](http://build.servo.org/builders/mac-rel-wpt4/builds/4197), [windows-msvc-dev](http://build.servo.org/builders/windows-msvc-dev/builds/7104) are reusable. Rebuilding only [mac-rel-wpt1](http://build.servo.org/builders/mac-rel-wpt1/builds/8433)...

bors-servo commented:
:sunny: Test successful - [android](http://build.servo.org/builders/android/builds/11399), [arm32](http://build.servo.org/builders/arm32/builds/11630), [arm64](http://build.servo.org/builders/arm64/builds/11676), [linux-dev](http://build.servo.org/builders/linux-dev/builds/11712), [linux-rel-css](http://build.servo.org/builders/linux-rel-css/builds/8608), [linux-rel-wpt](http://build.servo.org/builders/linux-rel-wpt/builds/8612), [mac-dev-unit](http://build.servo.org/builders/mac-dev-unit/builds/11765), [mac-rel-css1](http://build.servo.org/builders/mac-rel-css1/builds/4161), [mac-rel-css2](http://build.servo.org/builders/mac-rel-css2/builds/4190), [mac-rel-wpt1](http://build.servo.org/builders/mac-rel-wpt1/builds/8434), [mac-rel-wpt2](http://build.servo.org/builders/mac-rel-wpt2/builds/8495), [mac-rel-wpt3](http://build.servo.org/builders/mac-rel-wpt3/builds/4256), [mac-rel-wpt4](http://build.servo.org/builders/mac-rel-wpt4/builds/4197), [windows-msvc-dev](http://build.servo.org/builders/windows-msvc-dev/builds/7104)
Approved by: nox
Pushing d234aeb5af5450702211511995b9a19c57e38d52 to master...
